### DISCUSSION TITLE
RSS min update time 2min - why?

### DISCUSSION THREAD

**Discussion by jakobbg** [Novik/ruTorrent#2627] [General]
Hey! Thanks for a great web client!

Question: Is there a reason beyond reducing stress on web servers to have the update time on RSS feeds to be a minimum of 2 minutes? I'd like to set it to 1 minute.

**Comment by ranirahn** [comment]
What feed you want to update so often? That can get stuck to spam filters and then you get no updates for your rss at all. My update time is 1 hours and I have not seen that I have missed a single torrent that I am interested in.

**Comment by jakobbg** [comment]
😋 it is only a matter of being one of the primary seeders.

**Comment by ranirahn** [comment]
I am using for this kind of custom actions Flexget instead. If you want just to update RSS that you could do once per second with that one. Would not recommend, but you could.
